packages/api-sdk/src/resources/project/index.ts

```diff
@@ -109,6 +109,14 @@ class ProjectResource extends CrudResource<
     return data;
   }
 
+  public async getVersionsV2<P extends Models.VersionPlatformData>(id: Models.ProjectID): Promise<Models.Version<P>[]>;
+
+  public async getVersionsV2(id: Models.ProjectID, offset = 0, limit = 10): Promise<Models.Version<any>[]> {
+    const { data } = await this.fetch.get<Models.Version<any>[]>(`${this._getCRUDEndpoint(id)}/projectVersions?offset=${offset}&limit=${limit}`);
```

Comment:
would it be a good idea to make `offset` and `limit` named parameters?

```suggestion
  public async getVersionsV2<P extends Models.VersionPlatformData>(
    id: Models.ProjectID,
    { offset, limit } = { offset: 0, limit: 10 }
  ): Promise<Models.Version<P>[]> {
    const { data } = await this.fetch.get<Models.Version<P>[]>(`${this._getCRUDEndpoint(id)}/projectVersions?offset=${offset}&limit=${limit}`);
```

this also fixes a typings issue in the overload which prevents you from actually specifying an `offset` or `limit` value